src/ucp/wireup/select.c

```diff
@@ -1733,6 +1739,16 @@ ucp_wireup_construct_lanes(const ucp_wireup_select_params_t *select_params,
         if ((context->tl_mds[md_index].attr.cap.flags & UCT_MD_FLAG_NEED_RKEY) &&
             !(strstr(context->tl_rscs[rsc_index].tl_rsc.tl_name, "ugni"))) {
             key->rma_bw_md_map |= UCS_BIT(md_index);
+
+            if (key->err_mode == UCP_ERR_HANDLING_MODE_NONE) {
+                continue;
+            }
+
+            cap_flags = ucp_worker_iface_get_attr(worker, rsc_index)->cap.flags;
+            if (ucs_test_all_flags(cap_flags, UCT_IFACE_FLAG_CONNECT_TO_IFACE |
+                                              UCT_IFACE_FLAG_GET_ZCOPY)) {
+                key->rma_inv_md_map |= UCS_BIT(md_index);
```

Comment:
why do we need to save this in ep configuration key? can't we just invalidate all ep2iface rma_bw_lanes when we need to purge requests on the ep?